frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -76,3 +80,31 @@ export const toTitleCase = (title: string): string => {
 export const isModifiedEvent = (event: React.MouseEvent<HTMLElement>) => {
   return !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey);
 };
+
+const catalogItemCompare = (keyword: string, item: CatalogItem): boolean => {
```

Comment:
I think these utils are specific to catalog and should be co-located with it. If needed we can pull the generic catalog pieces out into `@console/shared` or maybe even SDK. @christianvogt WDYT?